File: Automation with API/Twillio Domain Avalability Checker/twilio_message_send.py
from twilio.rest import Client 
 
account_sid = 'account_sid' 
auth_token = 'auth_token' 
client = Client(account_sid, auth_token) 
 
message = client.messages.create( 
                              from_='whatsapp:+14155238886',  
                              body='Your Twilio code is 1238432',      
                              to='whatsapp:+4915128288055' 
                          ) 
 
print(message.sid)

File: Automation with API/Twillio Domain Avalability Checker/twilio_message_send.js
// npm install twilio
const twilio = require('twilio');

const accountSid = 'TWILLIO_SID';
const authToken = 'TWILLIO_TOKEN';

const client = new twilio(accountSid, authToken);
client.messages.create({
  from: 'whatsapp:+91232323334',
  body: 'Sent this message using code',
  to: 'whatsapp:+91232323323'
}).then(msg => console.log(msg.sid));